### PR TITLE
KW41Z IEEE802.15.4 dirver fix 

### PR DESCRIPTION
**Pull request by scpf19**
Added empty configure function to ieee802154_radio_api struct, to prevent HardFault.

### CODE REVIEW

File: drivers/ieee802154/ieee802154_kw41z.c [hunk 1]
```diff
@@ -1020,6 +1020,12 @@ static void kw41z_iface_init(struct net_if *iface)
 	ieee802154_init(iface);
 }
 
+static int kw41z_configure(struct device *dev, enum ieee802154_config_type type,
```
Comment (tbursztyka) [inline drivers/ieee802154/ieee802154_kw41z.c]: fix you commit title:  "drivers/ieee802154: Fix KW41Z driver fault"  and have a message explaining what was the issue.

File: include/arch/arm/cortex_m/scripts/linker.ld [hunk 2]
```diff
@@ -324,34 +324,6 @@ SECTIONS
 	} GROUP_DATA_LINK_IN(RAMABLE_REGION, RAMABLE_REGION)
 #endif
 
-#ifdef CONFIG_COVERAGE_GCOV
```
Comment (tbursztyka) [inline include/arch/arm/cortex_m/scripts/linker.ld]: What all below has to do with this driver fix?

**Review by tbursztyka — CHANGES_REQUESTED** [review]
Looks like you did not rebased properly your branch against master as it get a commit for the master branch merge, fix this.

**Comment by scpf19** [comment]
Sorry for not rebase my branch properly against master. I added a new pull request #16291, therefore closing this one.

**Pull request closed without merging**